### ISSUE TITLE
Compile and test borsh-rs in ledger, in a c source file

### ISSUE THREAD

**Issue by ailisp** [near/borsh#91]
After #84 or better after #85 , Compile either borsh-rs or borsh-c to a static library, link it with a c library, test it's working on a ledger device

**Comment by r-near** [comment]
Closing as part of issue triage. This tracking issue from the 2020 cross-language initiative was never acted on. Please open a new issue if still relevant.
